run/proc.sh

```diff
@@ -67,8 +67,11 @@ fi
 
 
 echo "VCF pre-processing"
+#run exome filters
+bcftools filter --regions-file /run/data_dependencies/filter_exonic/$REF_DIR.bed /out/vcf.gz -Oz -o /out/$1.recode.vcf.gz
```

Comment:
Can you add a parameter to determine whether the filter is applied (or not)

Reply:
I see